examples/docs/src/pages/themes/docs/configuration.mdx

```diff
@@ -238,6 +238,25 @@ export default {
 }
 ```
 
+## `defaultMenuCollapseLevel`
+
+Specifies the folder level at which the menu on the left is collapsed by default.
+
+**Type:** `number`
+
+**Default:** `1`
+
+**Example:**
+
+```jsx filename="theme.config.js"
+/**
+ * @type {import('nextra-theme-docs').DocsThemeConfig}
+ */
+export default {
+  defaultMenuCollapseLevel: 2
```

Comment:
this should be
```
sidebar: {
  defaultMenuCollapseLevel: 2
}
```

Reply:
The actual docs page is laid out a bit differently, so this example here doesn't really matter, right? I've updated the docs page as I think I understood you to request.